dask/dataframe/tests/test_shuffle.py

```diff
@@ -367,11 +367,7 @@ def test_set_index_with_explicit_divisions():
 
     ddf = dd.from_pandas(df, npartitions=2)
 
-    def throw(*args, **kwargs):
-        raise Exception()
-
-    with dask.config.set(get=throw):
-        ddf2 = ddf.set_index("x", divisions=[1, 3, 5])
```

Comment:
I think you should be able to go back to the original on these tests now that the compute is only called from within the `_calculate_division` function.

Reply:
Yes, you're right. I've reverted this.

Reply:
In `test_shuffle.py` I could remove it, but `gpuCI` keeps on failing unless I remove this. Doesn't like the `get` keyword and requires a specific `scheduler`.

Reply:
This makes me a little worried because this test is meant to ensure that no compute is happening in this case. What happens if you just change `get=` to `scheduler=`

Reply:
Replaced with `scheduler=`. Let's see what happens.